src/main/java/Ui/Superhero.java

```diff
@@ -0,0 +1,130 @@
+package Ui;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import WordList.ToLearn;
+import WordList.VocabList;
+import WordList.Vocabulary;
+import WordList.Deadline;
+import WordList.Event;
+
+public class Superhero {
+
+    VocabList inputList = new VocabList();
+    Scanner choice = new Scanner(System.in);
+    String input;
+    ArrayList<String> inputArray;
+
+    public void startVocabCheck() {
```

Comment:
Perhaps you can abstract some parts of this method since it is quite long (exceeds 30 lines).